rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1017,7 +1017,41 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
-    protected void processSVPFailure(Federation proposedFederation) {
+    protected void updateSvpState(Transaction rskTx) {
+        Optional<Federation> proposedFederationOpt = federationSupport.getProposedFederation();
+        if (proposedFederationOpt.isEmpty()) {
+            return;
+        }
+
+        Keccak256 rskTxHash = rskTx.getHash();
```

Comment:
nit: maybe this local var is redundant, we can directly call `.getHash()` from the param var. Feel free to leave as is.